packages/turf-voronoi/index.js

```diff
@@ -0,0 +1,45 @@
+import lineStringToPolygon from '@turf/linestring-to-polygon';
+import { lineString } from '@turf/helpers';
+import d3voronoi from 'd3-voronoi';
+
+function und3ify(polygon) {
+    return lineStringToPolygon(lineString(polygon.slice(0)));
+}
+
+/**
+ * Takes a FeatureCollection of points, and a bounding box, and returns a FeatureCollection
+ * of Voronoi polygons. 
+ * 
+ * The Voronoi algorithim used comes from the d3-voronoi package.
+ *
+ * @name voronoi
+ * @param {FeatureCollection<Point>|Feature[]<Point>} points to find the Voronoi polygons around.
```

Comment:
The input should only be `FeatureCollection<Point>`, an Array of Features isn't a valid GeoJSON and a single point doesn't would most likely not be valid for this type of operation.